src/iden3comm/packageManager.ts

```diff
@@ -126,18 +124,35 @@ export class PackageManager implements IPackageManager {
 
   /** {@inheritDoc IPackageManager.getMediaType} */
   getMediaType(envelope: string): MediaType {
-    let base64HeaderBytes: Uint8Array;
-
-    // full serialized
+    const error = new Error('Header is missing typ');
```

Comment:
typ is a header actually.
Better to change: missing header 'typ'in the envelope

Reply:
fixed